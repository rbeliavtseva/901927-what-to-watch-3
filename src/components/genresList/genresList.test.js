import React from "react";
import renderer from "react-test-renderer";
import GenresList from "./genresList.jsx";

const genres = [`Comedy`, `Action`, `Romance`, `Drama`];

it(`Render GenresList`, () => {
  const tree = renderer
    .create(<GenresList
      onFilterButtonClick={() => {}}
      genres={genres}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
