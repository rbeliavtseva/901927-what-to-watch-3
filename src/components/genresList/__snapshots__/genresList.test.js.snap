// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render GenresList 1`] = `
<ul
  className="catalog__genres-list"
>
  <li
    className="catalog__genres-item "
    onClick={[Function]}
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      Comedy
    </a>
  </li>
  <li
    className="catalog__genres-item "
    onClick={[Function]}
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      Action
    </a>
  </li>
  <li
    className="catalog__genres-item "
    onClick={[Function]}
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      Romance
    </a>
  </li>
  <li
    className="catalog__genres-item "
    onClick={[Function]}
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      Drama
    </a>
  </li>
</ul>
`;
